contracts/lbp/LBPManager.sol

```diff
@@ -105,66 +112,74 @@ contract LBPManager {
         uint256[] memory _endWeights,
         uint256[] memory _fees,
         bytes memory _metadata
-    ) external returns (address) {
+    ) external {
         require(!initialized, "LBPManager: already initialized");
         require(_beneficiary != address(0), "LBPManager: _beneficiary is zero");
 
         initialized = true;
         admin = msg.sender;
+        swapFeePercentage = _fees[0];
         feePercentage = _fees[1];
         beneficiary = _beneficiary;
         metadata = _metadata;
+        startTimeEndTime = _startTimeEndTime;
```

Comment:
In this [page](https://hackmd.io/UC3er9TsR5yx3_t3Z61IRg) you can read about why we changed it.

In short, we only initiate the LBPManager, and create the LBP in a different function call. Because of this we have to store the parameters in the state variables. Hope this answered the question?

Reply:
Hmm, yep, more or less, as I understand this is due to the reason that we want an admin to pay for the LBP creation?

Reply:
Oh, yep, now I remember, thx...)